packages/vscode-extension/src/debug/prerequisitesHandler.ts

```diff
@@ -257,12 +257,14 @@ export async function checkAndInstall(): Promise<Result<any, FxError>> {
     checkResults.push(accountResult);
 
     // local cert
-    const localCertResult = await resolveLocalCertificate(
-      localEnvManager,
-      `(${currentStep++}/${totalSteps})`
-    );
-    await progressHelper.end(Checker.LocalCertificate);
-    checkResults.push(localCertResult);
+    if (ProjectSettingsHelper.includeFrontend(projectSettings)) {
```

Comment:
Use `checkers.includes(Checker.LocalCertificate)` to keep the condition only one source of truth.

Reply:
Added